lib/src/supabase_auth.dart

```diff
@@ -6,6 +6,8 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:supabase_flutter/supabase_flutter.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'dart:io' show Platform;
```

Comment:
I believe importing this breaks the web support (at least throw an error when running on the web). To fix it, we must do the stub-real thing. We do the same with isolates, which arent supported on the web

https://github.com/flutter/plugins/tree/main/packages/path_provider/path_provider_windows/lib/src

Reply:
@bdlukaa 
That is what I thought first too, but it seems like as long as we are [checking for web first](https://github.com/supabase/supabase-flutter/pull/333/files#diff-f0d0a37c250b72c9bc959f740b43aa12057188574466716183e4a2d4ce42168aR109) it should be fine as other lines containing `Platform` don't get executed.

[Here](https://zapp.run/edit/flutter-z0a06ta0b06?entry=lib/main.dart&file=lib/main.dart) is an example app working on Flutter web!

Reply:
cool!